SurfaceTopography/Generic/ScaleDependentStatistics.py

```diff
@@ -67,6 +67,8 @@ def scale_dependent_statistical_property(self, func, n=1, scale_factor=None, dis
         Characteristic distances at which the derivatives are computed. If
```

Comment:
Could you clarify what the characteristic distance is ? If I understand the code correctly, it is scale_factor * pixel_size

Reply:
I've updated the docstring. Is this clearer?